src/raftstore/store/worker/apply.rs

```diff
@@ -110,6 +111,28 @@ pub enum ExecResult {
     VerifyHash { index: u64, hash: Vec<u8> },
 }
 
+struct ApplyContext {
+    pub wb: Option<WriteBatch>,
+    pub cbs: Vec<(Callback, RaftCmdResponse)>,
+}
+
+impl ApplyContext {
+    fn new() -> ApplyContext {
+        ApplyContext {
+            wb: Some(WriteBatch::new()),
+            cbs: vec![],
+        }
+    }
+
+    pub fn wb_mut(&mut self) -> &mut WriteBatch {
```

Comment:
There are a lot of places use `&mut WriteBatch` to indicate that will modify wb, should I follow the style?
https://github.com/pingcap/tikv/blob/master/src/raftstore/store/peer_storage.rs#L180
https://github.com/pingcap/tikv/blob/master/src/raftstore/store/peer_storage.rs#L171
https://github.com/pingcap/tikv/blob/master/src/raftstore/store/peer_storage.rs#L530
https://github.com/pingcap/tikv/blob/master/src/raftstore/store/peer_storage.rs#L565